components/layout_2020/flow/inline.rs

```diff
@@ -10,14 +10,14 @@ use crate::fragments::CollapsedBlockMargins;
 use crate::fragments::{AnonymousFragment, BoxFragment, Fragment, TextFragment};
 use crate::geom::flow_relative::{Rect, Sides, Vec2};
 use crate::positioned::{AbsolutelyPositionedBox, AbsolutelyPositionedFragment};
-use crate::sizing::{outer_inline_content_sizes_and_percentages, ContentSizes};
+use crate::sizing::{outer_inline_content_sizes_and_percentages, shrink_to_fit, ContentSizes};
```

Comment:
Nit: I prefer to call functions with paths qualified by their containing module.

Reply:
These are now methods (and so don’t need to be imported).